oauth2client/service_account.py

```diff
@@ -407,3 +407,38 @@ def create_scoped(self, scopes):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+    def create_delegated(self, sub):
+        """Create credentials that act as domain-wide delegation of authority.
+
+        Use the ``sub`` parameter as the subject to delegate on behalf of
+        that user.
+
+        For example::
+
+          >>> account_sub = 'foo@email.com'
+          >>> delegate_creds = creds.create_delegated(account_sub)
+
+        Args:
+            sub: string, An email address that this service account will
+                 act on behalf of (via domain-wide delegation).
+
+        Returns:
+            ServiceAccountCredentials, a copy of the current service account
+            updated to act on behalf of ``sub``.
+        """
+        new_kwargs = self._kwargs.copy()
```

Comment:
After the back-and-forth that the two of you had, how much would you dislike me if I asked for `new_kwargs = dict(self._kwargs)`? That would be what I'd prefer if they're all the same to you.